examples/website/scenegraph-layer/app.js

```diff
@@ -0,0 +1,151 @@
+/* global fetch, window */
+import React, {Component} from 'react';
+import {render} from 'react-dom';
+
+import {StaticMap} from 'react-map-gl';
+import DeckGL from '@deck.gl/react';
+import {ScenegraphLayer} from '@deck.gl/mesh-layers';
+
+import {GLBScenegraphLoader, GLTFScenegraphLoader} from '@luma.gl/addons';
+import {registerLoaders} from '@loaders.gl/core';
+
+registerLoaders([GLBScenegraphLoader, GLTFScenegraphLoader]);
```

Comment:
It would great if we could make these in a single loader. Do I just need to fix the multi-extension support in loaders.gl?

Reply:
Yes.